scripts/travis_script.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+if [ "${DISABLE_MOCK}" = "false" ]; then
```

Comment:
Could you use another env variable to decide with test to run? Reusing DISABLE_MOCK is a little confusing.